packages/core/package.json

```diff
@@ -96,7 +96,7 @@
         "@deriv/cfd": "^1.0.0",
         "@deriv/components": "^1.0.0",
         "@deriv/deriv-api": "^1.0.11",
-        "@deriv/deriv-charts": "1.3.5",
+        "@deriv/deriv-charts": "1.4.0",
```

Comment:
```suggestion
        "@deriv/deriv-charts": "^1.4.0",
```

Reply:
@akmal-deriv we don't want to automatically install the latest version, we want an exact one instead. :)